components/merkledb/src/db.rs

```diff
@@ -204,6 +204,10 @@ impl WorkingPatch {
     }
 
     /// Returns a mutable reference to the changes corresponding to a certain index.
+    ///
+    /// # Panics
+    ///
+    /// If two indices with the same `address` exist in `Fork` that uses this patch.
```

Comment:
It is not possible for two indices with the same `address` to exist within single `Fork`. Wouldn't it be better to reword like "If an index with the `address` already exists in ..."?